src/python/pants/backend/python/goals/coverage_py_integration_test.py

```diff
@@ -57,6 +57,7 @@ def test_add():
         python_tests(
             name="tests",
             dependencies=[":project"],
+            batch_compatibility_tag="default",
```

Comment:
These tests didn't _need_ to set a compatibility tag, right? Is this just to provide more test coverage?

Reply:
Correct - I wanted a quick check that I hadn't broken the expected behavior of coverage collection. If you want, I could probably turn `SOURCES` into a function w/ a `batched: bool` arg and then parameterize the tests for both batched and unbatched

Reply:
That could be good, yea...

Reply:
Nothing in the coverage code particularly cares about batching, but I wasn't sure if/how the output coverage files would differ and if the merging logic would need updated so I wanted to be sure.

I already went ahead and implemented the parameterized approach 😅 let me know if you want me to revert